lib/Manhattan.js

```diff
@@ -76,12 +76,7 @@ class Manhattan extends React.Component {
       return;
     }
 
-    const {
-      scaleMinusLogPValue,
-      scaleChromosomes,
-      focusChromosome,
-      chartIndex,
-    } = this.state;
+    const { scaleMinusLogPValue, scaleChromosomes, chartIndex } = this.state;
```

Comment:
Since `focusChromosome` is no longer obtained from the state, could we remove `focusChromosome` when setting the state in the constructor?

Reply:
Yes, good spot